tools/PI/DevHome.PI/ViewModels/AppDetailsPageViewModel.cs

```diff
@@ -121,4 +136,123 @@ private void RunAsAdmin()
             CommonHelper.RunAsAdmin(_targetProcess.Id, nameof(AppDetailsPageViewModel));
         }
     }
+
+    private void GetPackageInfo(ProcessDiagnosticInfo pdi)
+    {
+        if (pdi.IsPackaged)
+        {
+            AppInfo.IsPackaged = true;
+            ProcessPackageVisibility = Visibility.Visible;
+
+            var package = pdi.GetAppDiagnosticInfos().FirstOrDefault()?.AppInfo.Package;
+            if (package is not null)
+            {
+                if (package.Id is not null)
+                {
+                    AppInfo.PackageInfo.FullName = package.Id.FullName;
+                    var version = package.Id.Version;
+                    AppInfo.PackageInfo.Version = $"{version.Major}.{version.Minor}.{version.Build}.{version.Revision}";
+                }
+
+                AppInfo.PackageInfo.DisplayName = package.DisplayName;
+                AppInfo.PackageInfo.InstalledDate = package.InstalledDate.ToString(CultureInfo.CurrentCulture);
+                AppInfo.PackageInfo.InstalledPath = package.InstalledPath;
+                AppInfo.PackageInfo.Publisher = package.PublisherDisplayName;
+                AppInfo.PackageInfo.IsDevelopmentMode = package.IsDevelopmentMode;
+                AppInfo.PackageInfo.SignatureKind = $"{package.SignatureKind}";
+                AppInfo.PackageInfo.Status = GetPackageStatus(package);
+
+                List<string> dependencies = [];
```

Comment:
This can be a LINQ statement.  `List<string> dependencies = package.Dependencies.Select(x => x.Is.FullName).ToList()`

Reply:
Yes it could, but I don't see any advantage, and it might even be slower.